doc/developer_guide/developer_guide.md

```diff
@@ -14,6 +14,20 @@ poetry run nox -s build_language_container
 
 Installing the SLC ins described in the [AAF User Guide](../user_guide/user_guide.md#script-language-container-slc).
 
+## Update Generated Files
+
+AAF contains some generated files that are committed to git, including:
+* The amalgated Lua script [create_query_loop.sql](https://github.com/exasol/advanced-analytics-framework/blob/main/exasol_advanced_analytics_framework/resources/outputs/create_query_loop.sql)
+* The examples in the user guide
```

Comment:
is this still true

Reply:
Thanks - I removed the main instructions but forgot to update the bullet list. See next push.